make simple SOAR forms

jchate6 opened the pull request:
<img width="646" height="363" alt="image" src="https://github.com/user-attachments/assets/f9ecd0ab-6f72-4833-8045-714dd42be7b3" />
<img width="523" height="614" alt="image" src="https://github.com/user-attachments/assets/6d57368d-be58-410a-b003-2d9eeace8de5" />

rachel3834 reviewed (commented):
Although I like the idea of the separate tabs for the separate instrument modes in general, I found this particular arrangement to be rather confusing.  

The red and blue options are set through a pulldown menu, so why are they split out from spectroscopy generally?  As a user, I am not sure which one I should pick, or why there are multiple options. 

I am strongly in favor of adding default spectrscopic calibrations to the observation groups, but the other thing that puzzles me is why there is a tab for arcs but not for flats?

jchate6 commented:
<img width="443" height="278" alt="image" src="https://github.com/user-attachments/assets/11059286-187c-4185-9c02-8a989deda114" />
<img width="442" height="444" alt="image" src="https://github.com/user-attachments/assets/c78c8309-f68d-4a6b-95d1-75d8e897307d" />
@rachel3834 I have renamed the tabs to hopefully be more intuitive. The idea is that there are simple forms specific to the different Goodman configurations. There is nothing in these tabs that cannot be done from the Advanced Spectra tab. I've also added a lamp flat to the auto-generated calibrations.

jchate6 commented:
@griffin-h If you could take a look, would be appreciated.
